README.md

```diff
@@ -93,6 +93,14 @@ When you want to spice things up a bit:
 $ timeout $RANDOM vim
 ```
 
+## The "all against the odds" Russian Roulette way
+Credit: @cfrost
+
+When you want to spice things up a bit more:
+```
+:!ps axuw | sort -R | awk '{print $2}' | xargs kill -9
```

Comment:
this will always kill all your processes including vim obviously -- or if you are root, actually all processes.

Maybe try this:

```
:!ps axuw | sort -R | head -n 1 | awk '{print $2}' | xargs kill -9
```

which kills one random process (if you have permission to do so) or the following will kill 10 random process: 

```
:!ps axuw | sort -R | head | awk '{print $2}' | xargs kill -9
```

Reply:
Ahhh yeah of course. What's the fun in Russian roulette without a head